test/modules/TwinTester/TwinAllOperationsResultHandler.cs

```diff
@@ -72,11 +72,11 @@ public Task HandleReportedPropertyUpdateExceptionAsync(string failureStatus)
             return this.CallAnalyzer(failureStatus);
         }
 
-        async Task CallAnalyzer(string failureStatus)
+        async Task CallAnalyzer(string result)
```

Comment:
good change this would be very confusing

Reply:
✌️